### PR TITLE
Update to v5.23.5 and create a README with color scheme workaround info

### PR DESCRIPTION
**Pull request by vulongm**
Provides information on how to use KDE Plasma's color schemes and ascent color with the Breeze theme on flatpak using a workaround.

### CODE REVIEW

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
Is this overrulement operational for the 4th version of "GTK"?

**Review by BEEDELLROKEJULIANLOCKHART — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# org.gtk.Gtk3theme.Breeze
+### Workarounds
+- Using a custom color scheme (etc. Breeze Dark) and ascent color. Related issue [#3901](https://github.com/flatpak/flatpak/issues/3901).
+  ```sh
+  flatpak override --user --filesystem=xdg-config/gtk-3.0:ro
```
Comment (BEEDELLROKEJULIANLOCKHART) [inline README.md:5]: Should this additionally include "--filesystem=xdg-config/gtk-4.0:ro"?

Comment (vulongm) [reply README.md:5]: That wouldn't work as flatpak doesn't allow for custom Gtk4 themes yet and KDE doesn't store any color scheme information in that folder. A new package named ``org.gtk.Gtk3theme.Breeze`` might have to be created when flatpak adds the Gtk4 functionality as well.

Comment (mwleeds) [reply README.md:5]: @vulongm where did you get that information about Flatpak not allowing custom Gtk4 themes? I don't think anything in Flatpak itself would prevent `--filesystem=xdg-config/gtk-4.0:ro` from working, though I also don't know much about how themes work.

**Comment by Bleuzen** [comment]
Can the filesystem permission for `xdg-config/gtk-3.0:ro` be added to the manifest? Should be, no normal user will search in github repos for workarounds for basic things that should just work.

**Comment by hfiguiere** [comment]
no because this is not an app, so there is no permission associated to it.

**Comment by travier** [comment]
Let's split this into two PRs so that we can merge the version bump now and figure out the README later?

**Comment by vulongm** [comment]
> Let's split this into two PRs so that we can merge the version bump now and figure out the README later?

Yeah, I'll create two new PRs for those.

**Pull request closed without merging**